pr_agent/cli.py

```diff
@@ -33,43 +34,68 @@ def run(args=None):
                                                                   'describe', 'describe_pr',
                                                                   'improve', 'improve_code',
                                                                   'reflect', 'review_after_reflect'],
-                        default='review')
+                                                                   default='review')
     parser.add_argument('rest', nargs=argparse.REMAINDER, default=[])
     args = parser.parse_args(args)
     logging.basicConfig(level=os.environ.get("LOGLEVEL", "INFO"))
     command = args.command.lower()
-    if command in ['ask', 'ask_question']:
-        if len(args.rest) == 0:
-            print("Please specify a question")
-            parser.print_help()
-            return
-        print(f"Question: {' '.join(args.rest)} about PR {args.pr_url}")
-        reviewer = PRQuestions(args.pr_url, args.rest)
-        asyncio.run(reviewer.answer())
-    elif command in ['describe', 'describe_pr']:
-        print(f"PR description: {args.pr_url}")
-        reviewer = PRDescription(args.pr_url)
-        asyncio.run(reviewer.describe())
-    elif command in ['improve', 'improve_code']:
-        print(f"PR code suggestions: {args.pr_url}")
-        reviewer = PRCodeSuggestions(args.pr_url)
-        asyncio.run(reviewer.suggest())
-    elif command in ['review', 'review_pr']:
-        print(f"Reviewing PR: {args.pr_url}")
-        reviewer = PRReviewer(args.pr_url, cli_mode=True, args=args.rest)
-        asyncio.run(reviewer.review())
-    elif command in ['reflect']:
-        print(f"Asking the PR author questions: {args.pr_url}")
-        reviewer = PRInformationFromUser(args.pr_url)
-        asyncio.run(reviewer.generate_questions())
-    elif command in ['review_after_reflect']:
-        print(f"Processing author's answers and sending review: {args.pr_url}")
-        reviewer = PRReviewer(args.pr_url, cli_mode=True, is_answer=True)
-        asyncio.run(reviewer.review())
+    commands = {
+        'ask': _handle_ask_command,
+        'ask_question': _handle_ask_command,
+        'describe': _handle_describe_command,
+        'describe_pr': _handle_describe_command,
+        'improve': _handle_improve_command,
+        'improve_code': _handle_improve_command,
+        'review': _handle_review_command,
+        'review_pr': _handle_review_command,
+        'reflect': _handle_reflect_command,
+        'review_after_reflect': _handle_review_after_reflect_command
+    }
+    if command in commands:
+        commands[command](args.pr_url, args.rest)
     else:
         print(f"Unknown command: {command}")
         parser.print_help()
```

Comment:
**Suggestion:** Consider adding a default case to the commands dictionary to handle unknown commands. This will eliminate the need for the if-else statement checking if the command is in the dictionary.
```suggestion
    def _handle_unknown_command(pr_url: str, rest: list):
        print(f"Unknown command: {command}")
        parser.print_help()

    commands = {
        'ask': _handle_ask_command,
        'ask_question': _handle_ask_command,
        'describe': _handle_describe_command,
        'describe_pr': _handle_describe_command,
        'improve': _handle_improve_command,
        'improve_code': _handle_improve_command,
        'review': _handle_review_command,
        'review_pr': _handle_review_command,
        'reflect': _handle_reflect_command,
        'review_after_reflect': _handle_review_after_reflect_command,
        'default': _handle_unknown_command
    }

    commands.get(command, commands['default'])(args.pr_url, args.rest)
```